auth_callout/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "version": "0.0.0",
+  "name": "root",
+  "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
+  "devDependencies": {
```

Comment:
check the versions and figure out how we are going to update them automatically. Does dependabot handles this?

Reply:
Done